Quantum-35/Ride_My_Way
# Ft develop user page #158455361

**What does this PR do?**
Creates user Page where users can do the following:
```

1. Can view all available ride offers
2. Can see the details of a ride offer and request to join the ride. E.g What time
   the ride leaves, where it is headed e.t.c
3.See the history of the rides they joined 
```

**Description of Task to be completed**
 Make the ride history page responsive
**How should I test it?**
Clone this repository to a local folder
Navigate to the cloned repo
git checkout Ft develop user page #158455361
Open index.html using your favorite browser
Then click on the user Profile to see profile
**What PT stories?**
[Ft develop user page #158455361] https://www.pivotaltracker.com/n/projects/2179783

**Screenshot for the user Profile**
![userpage-1](https://user-images.githubusercontent.com/39088834/41637888-112691ec-745f-11e8-981e-248a81c2ae3c.jpg)
**Screenshot for the User profile with Details Model Opened**
![userpage-2](https://user-images.githubusercontent.com/39088834/41637903-29ede414-745f-11e8-821f-dd7cc9b538d5.jpg)
**Screenshot for the users Ride history**
![userpage-4](https://user-images.githubusercontent.com/39088834/41638176-91d05aac-7460-11e8-876b-47e499374c55.jpg)
